app/models/team.rb

```diff
@@ -5,14 +5,39 @@ class Team < ActiveRecord::Base
   validates :full_Id, presence: true, uniqueness: true
   validates :name, presence: true
 
-  def self.make(team, summoner)
-    roster = team["roster"]["memberList"]
-    new_team = Team.find_or_create_by(full_Id: team["fullId"],
-                                  name: team["name"],
-                                  tag: team["tag"])
-    roster.each do |player|
-      summoner = Summoner.add(player["playerId"])
-      Roster.make_connection(new_team, summoner)
+  def self.make(team_array, summoner)
+    player_ids_to_query = []
+    team_array.each do |t|
+      roster = t["roster"]["memberList"]
+      roster.each do |r|
+        player_ids_to_query << r['playerId']
+      end
+    end
+
+    File.open('temp.json', 'w') do |f|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.